Updated implementing custom error pages article (#7249)
umbraco/UmbracoDocs: 15/umbraco-cms/tutorials/custom-error-page.md

## 404 Errors

> A 404 error occurs when a requested page cannot be found—typically because the content has been deleted, the URL path has changed, or the visitor typed an invalid path. In Umbraco, you can create and configure custom 404 pages using content from the backoffice.

Author: ```suggestion
A 404 error occurs when a requested page cannot be found, usually due to deleted content, a changed URL, or an invalid path. In Umbraco, you can create and configure custom 404 pages using content from the backoffice.
```
